component-models.json

```diff
@@ -1083,27 +1087,31 @@
         "component": "text",
         "valueType": "string",
         "name": "surname",
+        "required": true,
```

Comment:
Peofile fields will be optional as these will be configured only on profile pages.

Reply:
removed "required" from people fields